### PR TITLE
Features/#7 change meetup api with v3

### PR DESCRIPTION
**Pull request by mertceyhan**

### CODE REVIEW

File: meetupapi/src/main/java/com/gdgistanbul/api/MeetupApi.kt [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 interface MeetupApi {
 
-    @GET("events")
-    suspend fun getEvents(@Query("has_ended") hasEnded: Boolean = true): List<Event>
+    @GET("{url_name}/events")
+    suspend fun getEvents(@Path("url_name") group: String = "GDGIstanbul"): List<Event>
 
-    @GET("events/{id}/attendance")
-    suspend fun getAttendees(@Path("id") id: String, @Query("order") order: String = "name")
+    @GET("{url_name}/events/{id}/attendance")
```
Comment (oshamahue) [inline meetupapi/src/main/java/com/gdgistanbul/api/MeetupApi.kt:12]: add url_name path parameter to the parameters also there is no group on the path.

File: meetupapi/src/main/java/com/gdgistanbul/api/MeetupApi.kt [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
+    suspend fun getAttendees(
+        @Path("group") group: String = "GDGIstanbul",
+        @Path("id") id: String,
+        @Query("order") order: String = "name"
+    ): List<Attendance>
 
 
-    @GET("events/{id}/attendance")
-    suspend fun getAttendenceTaking(@Field("member") id: String, @Field("status") status: String )
+    @POST("{url_name}/events/{id}/attendance")
```
Comment (oshamahue) [inline meetupapi/src/main/java/com/gdgistanbul/api/MeetupApi.kt:20]: add url_name path parameter to the parameters

File: model/src/main/java/com/gdgistanbul/model/Model.kt [hunk 3]
```diff
@@ -4,38 +4,159 @@ import com.squareup.moshi.Json
 import com.squareup.moshi.JsonClass
 
 @JsonClass(generateAdapter = true)
-data class EventContext(val host: Boolean)
+data class Event(
+    @Json(name = "created")
```
Comment (oshamahue) [inline model/src/main/java/com/gdgistanbul/model/Model.kt]: @Json annotation is not needed if the json name and parameter name are same. so remove the ones that are not snake case.

**Review by oshamahue — CHANGES_REQUESTED** [review]

File: meetupapi/src/main/java/com/gdgistanbul/api/MeetupApi.kt (hunk 2 above)
Comment (mertceyhan) [reply meetupapi/src/main/java/com/gdgistanbul/api/MeetupApi.kt:20]: fixed

File: meetupapi/src/main/java/com/gdgistanbul/api/MeetupApi.kt (hunk 1 above)
Comment (mertceyhan) [reply meetupapi/src/main/java/com/gdgistanbul/api/MeetupApi.kt:12]: fixed

File: model/src/main/java/com/gdgistanbul/model/Model.kt (hunk 3 above)
Comment (mertceyhan) [reply model/src/main/java/com/gdgistanbul/model/Model.kt]: fixed

**Review by oshamahue — APPROVED** [review]

**Pull request merged**